### PR TITLE
[T3A4][W11-A2]Fan Weiguang

### PR DESCRIPTION
**Pull request by fanwgwg**
Ready for review

### CODE REVIEW

File: src/seedu/addressbook/Main.java [hunk 1]
```diff
@@ -79,6 +79,12 @@ private void exit() {
     private void runCommandLoopUntilExitCommand() {
         Command command;
         do {
+        	try {
+				this.storage.checkFileExist();
+			} catch (FileNotExistException e) {
+				ui.showToUser(e.getMessage());
+				exit();
+			}
```
Comment (cmkumar87) [inline src/seedu/addressbook/Main.java]: indentations are off.

File: src/seedu/addressbook/storage/StorageFile.java [hunk 2]
```diff
@@ -69,6 +78,17 @@ public StorageFile(String filePath) throws InvalidStorageFilePathException {
...
+    
+
+    /**
+     * Check whether the storage file exists at the path
+     * @throws FileNotExistException
+     */
+    public void checkFileExist() throws FileNotExistException{
+    	if(!new File(path.toString()).exists()) {
+    		throw new FileNotExistException("Storage file doesn't exist");
+    	}
```
Comment (cmkumar87) [inline src/seedu/addressbook/storage/StorageFile.java]: So what happens in this method if file does exist? That is, what is the normal control flow?

**Comment by cmkumar87** [comment]
@fanwgwg Please review comments and close the PR. We are now advised to follow this protocol.

File: src/seedu/addressbook/storage/StorageFile.java (hunk 2 above)
Comment (fanwgwg) [inline src/seedu/addressbook/storage/StorageFile.java]: Oh yes maybe I should have changed the return type to boolean.

**Comment by fanwgwg** [comment]
Comments reviewed. Thank you very much.

**Pull request closed without merging**